GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -19,6 +23,7 @@ namespace gdjs {
         : null;
       this._runtimeScene = runtimeScene;
       this._pixiContainer = new PIXI.Container();
+      this._renderedObjectsPoints = [];
```

Comment:
Use objects for maps, not arrays. IDs are not necessarily in order, some IDs can be skipped, there are not guarantees, so if an ID is one day 100004, it means this array would have to be resized to contain 100004 elements. An object is the best way to associate things to a key.